packages/config-types/scripts/generate.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import fs from 'fs-extra';
+import { compile } from 'json-schema-to-typescript';
+import path from 'path';
+import semver from 'semver';
+
+let version = process.argv[2];
+
+if (!version) {
+  const packageJSON = require('../package.json');
+  // @ts-ignore
+  version = semver.parse(packageJSON.version).major;
+  console.log('Using package version: ' + version);
+}
+let parsedVersion = version;
+if (parsedVersion !== 'unversioned') {
+  parsedVersion += '.0.0';
+} else {
+  parsedVersion = parsedVersion.toUpperCase();
+}
+
+(async () => {
+  const {
+    data: { data },
+  } = await axios.get(`http://exp.host/--/api/v2/project/configuration/schema/${parsedVersion}`);
```

Comment:
it would be nice if this supported a local path as well, to make it easier to test schema changes. i'd like to be able to run `yarn generate --schema ~/universe/server/www/xdl-schemas/39.0.0-schema.json`

Reply:
done!